src/Http/Controllers/MediaController.php

```diff
@@ -14,11 +14,13 @@ class MediaController extends Controller
      */
     public function store(): string
     {
+        $diskName = config('canvas.storage_disk');
```

Comment:
I'd like to keep this file as it was, as well as removing the IDE helper. It's overkill to me.